app/services/idv/proofer.rb

```diff
@@ -68,7 +68,7 @@ def mock_vendors
 
       def require_mock_vendors_if_enabled
         return unless mock_fallback_enabled?
-        Dir[Rails.root.join('lib', 'proofer_mocks', '*')].each { |file| require file }
+        Dir[Rails.root.join('lib', 'proofer_mocks', '*')].sort.each { |file| require file }
```

Comment:
was this a cop? that's silly :[

Reply:
It is, but I think it's ok. The require order being non-deterministic can lead to some weird situations.